specification/gp-connect-access-record-fhir.yaml

```
@@ -1,13 +1,20 @@
 # This is an OpenAPI Specification (https://swagger.io/specification/)
 # for gp-connect-access-record-fhir owned by NHS Digital (https://digital.nhs.uk/)
 openapi: '3.0.0'
+x-spec-publication:
+  operation-order:
+    - operations:
+      - method: POST
+        path: /FHIR/STU3/$access-record-structured
```

this needs to be an operation on /Patient/, consistent with endpoint below on lne 255
```suggestion
        path: /FHIR/STU3/Patient/$access-record-structured
```